GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -252,6 +252,14 @@ extension SecureConversations {
         var engagementAction: EngagementViewModel.ActionCallback?
         var engagementDelegate: EngagementViewModel.DelegateCallback?
 
+        // Used to check whether custom card contains interactable metadata.
+        var isInteractableCard: ((MessageRenderer.Message) -> Bool)?
+        // Used to check whether custom card should be hidden.
```

Comment:
does it make sense to add these comments to doc (just add one more slash)?

Reply:
Fixed 7a289ee6a4529fbf7ec93aafa69f51da69af8020